draft.md

```diff
@@ -143,7 +143,8 @@ An [AppImage] which conforms to the type 2 image format:
 * **MUST** contain a file named `AppRun` in its root directory
 * **SHOULD** contain a [payload application] that is ultimately executed when the [AppImage] is executed
 * **SHOULD** contain exactly one `$APPNAME.desktop` file in its root directory with `$APPNAME` being the name of the [payload application]
-* **MAY** contain an `$APPNAME.png` file in its root directory with `$APPNAME` being the name of the [payload application] as set in the `Icon=` key of the `$APPNAME.desktop` file. If present, this icon **SHOULD** be given preference as the icon being used to represent the [AppImage]. The icon **SHOULD** be a PNG with 256x256 or 512x512 pixels
+* **MAY** contain icon files below `usr/share/icons/hicolor` following the  [Icon Theme Specification](https://standards.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html) for the icon identifier as set in the `Icon=` key of the `$APPNAME.desktop` file. If present, these icon files **SHOULD** be given preference as the icon being used to represent the [AppImage].
+* **MAY** contain an `$APPICON.svg`, `$APPICON.svgz` or `$APPICON.png` file in its root directory with `$APPICON` being the icon identifier as set in the `Icon=` key of the `$APPNAME.desktop` file. If present and no icon files matching the icon identifier present below `usr/share/icons/hicolor`, this icon **SHOULD** be given preference as the icon being used to represent the [AppImage]. If a PNG file, the icon **SHOULD** be of size 256x256, 512x512, or 1024x1024 pixels.
```

Comment:
We should highlight that PNG files are the ones which do work, the SVG stuff isn't supported by libappimage (might be changed by @azubieta at some point).

Reply:
The spec should not be written based on what some concrete implementations happen to do at the moment; it should describe the target picture. I think it is quite clear that throughout the Linux desktop, png, svg, svgz, and xpm (legacy) have to be supported.